### ISSUE TITLE
Could not find iPhone 6 simulator

### ISSUE THREAD

**Issue by ovaisahmed43** [facebook/react-native#21571]
**Code I am executing:**
`react-native run-ios`

**Error:**
```
Owaiss-Mac:pdm owaisahmed$ react-native run-ios
Found Xcode project pdm.xcodeproj

Could not find iPhone 6 simulator

Error: Could not find iPhone 6 simulator
    at resolve (/Users/owaisahmed/Documents/react-native-projects/pdm/node_modules/react-native/local-cli/runIOS/runIOS.js:149:13)
    at new Promise (<anonymous>)
    at runOnSimulator (/Users/owaisahmed/Documents/react-native-projects/pdm/node_modules/react-native/local-cli/runIOS/runIOS.js:134:10)
    at Object.runIOS [as func] (/Users/owaisahmed/Documents/react-native-projects/pdm/node_modules/react-native/local-cli/runIOS/runIOS.js:106:12)
    at Promise.resolve.then (/Users/owaisahmed/Documents/react-native-projects/pdm/node_modules/react-native/local-cli/cliEntry.js:117:22)
```

**Result of `react-native info`:**
```
  React Native Environment Info:
    System:
      OS: macOS 10.14
      CPU: x64 Intel(R) Core(TM) i5-2520M CPU @ 2.50GHz
      Memory: 32.16 MB / 3.97 GB
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.11.0 - /usr/local/bin/node
      Yarn: 1.10.1 - /usr/local/bin/yarn
      npm: 6.4.1 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    SDKs:
      iOS SDK:
        Platforms: iOS 12.1, macOS 10.14, tvOS 12.1, watchOS 5.1
    IDEs:
      Android Studio: 3.2 AI-181.5540.7.32.5014246
      Xcode: 10.1/10O35n - /usr/bin/xcodebuild
    npmPackages:
      react: 16.5.0 => 16.5.0 
      react-native: 0.57.1 => 0.57.1 
    npmGlobalPackages:
      react-native-cli: 2.0.1
```

**PS:** I am new to Mac OS, so problem might be very basic.
Thanks in advance for the Help!

**Comment by TalhaBinAsif** [comment]
I love  you

**Comment by TalhaBinAsif** [comment]
I am facing same issue.

**Comment by kelset** [comment]
Actually, this is already fixed via https://github.com/facebook/react-native/commit/6d09df5b726ac951417b87a49bc345ebc9142951

**Comment by ploureiro** [comment]
This Fix also correct all simulators startup:
- Xcode 10 simulator list Fix facebook/react-native@1031872

**Comment by elzii** [comment]
I'm having same issue on macOS Mojave after manually downgrading XCode & Command Line Tools to 9.4 [from here](https://developer.apple.com/download/more/).

`xcrun --version` gives `43` and `xcrun simctl list --json devices` shows then newer `isAvailable` syntax.

This is after removing any standalone tools via `sudo rm -rf /Library/Developer/CommandLineTools`

I can still build manually from XCode, but can't start via `react-native run-ios`. 

All that to say, @kelset pretty please cherry pick soon :)
